worker/create_batches_worker.go

```diff
@@ -116,10 +116,13 @@ func (b *CreateBatchesWorker) processBatch(ids *[]string, job *model.Job) {
 		cm.Write(zap.Int("usersInBatch", numUsersFromBatch))
 	})
 
-	b.sendBatches(*usersFromBatch, job)
+	if numUsersFromBatch != 0 {
+		b.sendBatches(*usersFromBatch, job)
 
-	b.updateTotalBatches(1, job)
-	b.updateTotalTokens(numUsersFromBatch, job)
+		b.updateTotalUsers(job, numUsersFromBatch)
```

Comment:
If we are only updating when > 0, what happens when it's zero? Is it the default value ?

Reply:
I'm not sure I got the question, but what happens here is that it's going to be zero unless there's it finds a user with the ID present in the file. If there are no users, it won't alter the total batches, users and tokens which will remain being zero, later one is used to check if the `completedAt` field should be set

Reply:
> it's going to be zero unless
So zero is the default value for this field, correct?

Reply:
Yes